internal/webhook/etcdcomponentprotection/handler.go

```diff
@@ -164,6 +173,13 @@ func isObjManagedByDruid(objMeta metav1.ObjectMeta) bool {
 	return hasLabel && managedBy == druidv1alpha1.LabelManagedByValue
 }
 
+func isRuntimeComponent(requestGK schema.GroupKind) bool {
```

Comment:
what does `isRuntimeComponent` mean actually?

Reply:
Whether the current resource for which the webhook has been triggered is a runtime component for the etcd cluster. I'll add a docstring to make this clear.